crates/bevy_ecs/src/system/query.rs

```diff
@@ -975,6 +975,44 @@ where
         self.state
             .is_empty(self.world, self.last_change_tick, self.change_tick)
     }
+
+    /// Returns `true` if the given [`Entity`] matches the query.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// # #[derive(Component)]
+    /// # struct Player;
+    /// # #[derive(Component)]
+    /// # struct Alive;
+    /// fn check_if_alive(
+    ///     player_query: Query<Entity, With<Player>>,
+    ///     alive_query: Query<(), With<Alive>>,
+    /// ) {
+    ///     if let Ok(player) = player_query.get_single() {
+    ///         if alive_query.contains(player) {
+    ///             println!("The player is alive.");
+    ///         } else {
+    ///             println!("The player is dead!");
+    ///         }
+    ///     }
+    /// }
+    /// # check_if_alive.system();
+    /// ```
+    #[inline]
+    pub fn contains(&self, entity: Entity) -> bool {
+        unsafe {
+            self.state.get_unchecked_manual(
```

Comment:
At the moment, this unsafe block is identical to the one in `get`, apart from it's missing the safety comment.

Please just replace the body with `self.get(entity).is_ok()`.

I think that a more optimised version is possible, but it would be challenging to review due to the amount of unsafety required.